src/templates/zone.ejs

```diff
@@ -1,5 +1,9 @@
 <div class="overlay" data-zone="<%= zone.id %>" style="display:none">
     <div class="cards">
+        <div class="audio-player">
+            <span class="close-deck">&times;</span>
```

Comment:
Consider moving this outside of the `.audio-player` DOM since it's a general card element.